frontend/src/app/main-table/volumes-table/volume-table.component.scss

```diff
@@ -0,0 +1,86 @@
+.card {
```

Comment:
This is copied from another component. Move it up and reference it from both places.